components/pkg-export-helm/src/deps.rs

```diff
@@ -47,7 +47,13 @@ impl Deps {
     }
 
     pub fn generate(&mut self, write: &mut Write) -> Result<()> {
-        let out = self.into_string()?;
+        // TODO: Until this Helm issue is resolved or has a decent workaround, let's not actually
```

Comment:
> let's not actually

skip adding the habitat operator as a dependancy:

Reply:
Fixed with a forced push.